decidim-budgets/app/controllers/decidim/budgets/projects_controller.rb

```diff
@@ -49,8 +49,8 @@ def default_filter_params
         {
           search_text_cont: "",
           with_any_status: default_filter_status_params,
-          with_any_scope: default_filter_scope_params,
```

Comment:
Since we are removing this, in all the places, maybe worth removing also `default_filter_category_params` and `default_filter_scope_params` method definitions: 
https://github.com/decidim/decidim/blob/bff1f56fc3edc85b8e853d0bc2f8b3e19323d665/decidim-core/app/controllers/concerns/decidim/filter_resource.rb#L73-L87

Reply:
Done in 030c72d7aa7b3af899e7a34fd718956139633f9b . Let me know if that's good for you or you wanted anything else. 

I've also removed a concern that we weren't using anymore (f85099a)